### DISCUSSION TITLE
Help/advice for fixing typos in HTML elements reference docs

### DISCUSSION THREAD

**Discussion by Kernix13** [mdn/mdn-community#303] [Content] [answered]
Hello, this is my first interaction with MDN here on GitHub. There are a lot of markdown files with a lot of links to other markdown files or MDN pages. I could spend all day reading them to try and figure out the best way to make or suggest changes for some typos I found, but I'm hoping someone here can give me a list of bullet points to follow.

I am going through all the HTML elements and making notes on them. I found some typos on some of the element pages. How would I go about making a change?

1. Forking then making the change and creating a pull request?
2. Opening an issue about the typo with links?
3. Other?

Example:

Page with typo: [<link>: The External Resource Link element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link)

Text with typo:

> The rel stands for "relationship", and is probably one of the key features of the <link> element — the value denotes how the item being linked to is related to the containing document

My suggestion is to remove the word `probably`. Is it one of the key features or not? I would say it is, but the use of the word `probably` makes it sound like "we're not sure" or it's an opinion by the writer of the doc. 

Whether or not admins think it should be changed, what are the steps submitting it as a potential change to the doc?

**Comment by bsmth** [comment, marked as answer]
Hi @Kernix13, thanks a lot for asking.

>How would I go about making a change? Forking then making the change and creating a pull request?

This is how to contribute changes, indeed. There's a section in our Contributing.md document that covers [simple changes](https://github.com/mdn/content/blob/main/CONTRIBUTING.md#simple-changes) that you can do via the GitHub UI. If you're doing this in multiple files, forking the repo and batching your changes in a single pull request is better. You can find more information in the [contributing.md](https://github.com/mdn/content/blob/main/CONTRIBUTING.md#forking-and-cloning-the-repository).

I hope that answers your question and thanks for contributing to the project :)

**Reply by Kernix13** [reply]
Got it. I forked and cloned the `content` repo. I am finding small typos as I go through the HTML element pages. I am creating a branch for each element page then pushing them individually. I wouldn't want to batch them together in case 1 was not accepted. I'll read through those links - thanks!

**Reply by bsmth** [reply]
Thanks @Kernix13, looking forward to it!

**Comment by Josh-Cena** [comment]
> is probably one of the key features of the element

I'm reluctant to call this a typo, because "key feature" is indeed a writer's opinion, not a fact. It's fair for different people to have different ideas about what the key features are, just that most may agree that `rel` would be one of them. It sounds quite natural to me to put a "probably" there.

**Reply by Kernix13** [reply]
Sorry for the late reply. My point is "probably" should be removed, not as a typo but as a word that does not belong there. The phrase "key feature" is fine.

**Reply by Josh-Cena** [reply]
Yes, I'm saying "probably" is necessary here, to emphasize the writer's uncertainty. If it's assertive like "this is one of the key features", that's an open invitation for arguments over what the set of key features are, which is not what we want in technical documentation. Making it less assertive is a deliberate choice to soften the writer's opinion.

**Reply by Kernix13** [reply]
Ok, I wasn't aware of that.
